app/js/lib/reducers/form_pagination.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var formPage = require( './form_pages' );
+
+function currentPage( page, pageLength, action ) {
+	var defaultPage = page || ( pageLength > 0 ? 0 : -1 );
```

Comment:
Will do the wrong thing if `page` is `0`. Please add tests for `0`, `-1`, `1` cases when doing things like these.